sematic/config/settings.py

```diff
@@ -137,6 +168,40 @@ def get_setting(self, var: AbstractSettingsVar, *args) -> str:
 
         raise MissingSettingsError(var, self.cli_command)
 
+    def get_plugin_settings(self, var: AbstractSettingsVar) -> PluginsSettings:
```

Comment:
For server deployment, this setup means we will need to do either one or both of:
- refactor the helm chart to mount the settings as a file rather than as env vars
- create some way to override nested settings via env var (ex: `plugins.foo.bar` maps to the env var `PLUGINS___FOO___BAR` or something like that)

cc: @snoshy

Reply:
I actually say somewhere in this PR that plugin settings must have unique names across the entire global namespace so that we don't have such hacks to map env vars to paths in the YAML files.

Reply:
Or we design a cli command like
```
sematic settings set --merge_yaml my patch.yaml
```

so it's up to the users to create the `patch.yaml` using whatever way to source whatever env vars he or she likes.